examples/traitsui_based/img_viewer.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import division
+
+from chaco.api import (
+    ArrayDataSource, DataRange1D, LinearMapper, OverlayPlotContainer, PlotAxis,
+    add_default_axes, add_default_grids)
+from chaco.tools.api import PanTool, ZoomTool
+
+from mapping.chaco.map import Map
+from mapping.chaco.null_renderer import NullPlotRenderer
+from mapping.enable.img_tile_manager import ImageTileManager
+
+
+def build_formatter(maxval):
+    def convert(x):
+        return '{:.0f}'.format(x * maxval)
+    return convert
+
+
+def create_plot_component(lod_dir):
+    index = ArrayDataSource()
+    index_range = DataRange1D(low=0, high=1)
+    index_range.add(index)
+    index_mapper = LinearMapper(range=index_range)
+
+    value = ArrayDataSource()
+    value_range = DataRange1D(low=0, high=1)
+    value_range.add(value)
+    value_mapper = LinearMapper(range=value_range)
+
+    plot = NullPlotRenderer(index=index, value=value,
+                            index_mapper=index_mapper,
+                            value_mapper=value_mapper,
+                            orientation='h', border_visible=True)
```

Comment:
curious.. what is the purpose of the `NullRenderer` ? Is it just to be able to add axis and such ?

Reply:
Yes, it's a way to get all the trimmings of a chaco plot (axes, title) without obscuring the canvas in any way.